config/routes.rb

```diff
@@ -58,6 +58,11 @@
 
   root 'v0/example#index', module: 'v0'
 
+  # route for testing with ID.me locally without front-end vets-website repo
+  if Rails.env.development?
+    get '/auth/login/callback', to: 'v0/sessions#saml_callback', module: 'v0'
```

Comment:
if you notice I made a corresponding comment on the README about the port: https://github.com/department-of-veterans-affairs/vets-api/pull/197/files#diff-04c6e90faac2675aa89e2176d2eec7d8R105